.github/workflows/scripts/generate-playground-blueprint.js

```diff
@@ -0,0 +1,104 @@
+const https = require( 'https' );
+
+const generateWordpressPlaygroundBlueprint = ( prNumber, artifactUrl ) => {
+	const defaultSchema = {
+		meta: {
+			title: `Remote Data Blocks - PR #${ prNumber }`,
+			description: `Installs remote-data-blocks plugin PR #${ prNumber } to WordPress Playground`,
+			author: 'WordPress VIP',
+			categories: [ 'Content' ],
+		},
+
+		features: {
+			networking: true,
+		},
+
+		landingPage: '/wp-admin/admin.php?page=remote-data-blocks-settings',
+
+		login: true,
+
+		preferredVersions: {
+			php: '8.2',
+			wp: 'latest',
+		},
+
+		steps: [
+			{
+				step: 'setSiteOptions',
+				options: {
+					blogname: `Remote Data Blocks - PR #${ prNumber }`,
+					blogdescription: `Explore the remote-data-blocks plugin PR #${ prNumber } in a WordPress Playground`,
+				},
+			},
+			{
+				step: 'defineWpConfigConsts',
+				consts: {
+					USE_PLAYGROUND_CORS_PROXY: true,
+				},
+			},
+			{
+				step: 'installPlugin',
+				pluginData: {
+					caption: 'Installing Remote Data Blocks',
+					resource: 'url',
+					url: `${ artifactUrl }`,
+				},
+				options: {
+					activate: true,
+					targetFolderName: 'remote-data-blocks',
+				},
+			},
+		],
+	};
+
+	return defaultSchema;
+};
+
+async function run( { github, context, core, artifactUrl } ) {
+	const commentInfo = {
+		owner: context.repo.owner,
+		repo: context.repo.repo,
+		issue_number: context.issue.number,
+	};
+
+	const comments = ( await github.rest.issues.listComments( commentInfo ) ).data;
+	let existingCommentId = null;
+
+	for ( const currentComment of comments ) {
+		if (
+			currentComment.user.type === 'Bot' &&
+			currentComment.body.includes( 'Test using WordPress Playground' )
+		) {
+			existingCommentId = currentComment.id;
+			break;
+		}
+	}
+
+	const defaultSchema = generateWordpressPlaygroundBlueprint( context.issue.number, artifactUrl );
+
+	const url = `https://playground.wordpress.net/#${ JSON.stringify( defaultSchema ) }`;
```

Comment:
Do we need to escape the stringified JSON so it doesn't break the markdown URL tag?

Reply:
Yip, that is the reason. Once it's escaped, it won't be broken. I just haven't done it yet as I've been trying to fix the playground link to the zip file first.

Note: Usually, I'd keep such PRs as draft PRs but this one made sense as a non-draft PR so it could be tested properly.

Reply:
Edit: Think escaping cause some weird quirks with playground that has to be dug into.